src/Polytopes/Polyhedron/properties.jl

```diff
@@ -836,13 +804,11 @@ false
 """
 isfulldimensional(P::Polyhedron) = pm_polytope(P).FULL_DIM
 
-"""
-    f_vector(P)
-
-Compute the vector`(f_1,f_2,...,f_(dim(P)-1))` where `f_i` is the number of faces of `P` of dimension `i`.
+@doc Markdown.doc"""
+    f_vector(P::Polyhedron)
 
-# Arguments
-- `P::Polyhedron`: A polyhedron.
+Compute the vector $(f₁,f₂,...,f_{(dim(P)-1))$` where $f_i$ is the number of
```

Comment:
I am not sure if we want to allow use of unicode like this or not. Personally, I don't mind, but others might... CC @fieker @thofma 
```suggestion
Compute the vector $(f_1,f_2,...,f_{(dim(P)-1))$` where $f_i$ is the number of
```